src/app/services/auth.service.ts

```diff
@@ -39,6 +39,15 @@ export class AuthService {
                     .replace('__scopes__', config.scopes);
                 this.oAuthUserUrl = config.userInfoUrl;
                 this.oAuthUserNameField = config.userInfoNameField;
+
+                this.token = localStorage.getItem('access_token');
+                if (this.token) {
+                    this.authenticated = true;
```

Comment:
I believe there is a problem here, though I like what you are doing in general.  I don't think you can set `this.authenticated` to true unless a) there is an expires value in local storage and b) that value has not passed already.  We can't set auth to true if there's no expires value, and we can't set it to true if that expires value has already expired.  If you can fix that up, I'll accept the new merge request.